### ISSUE TITLE
archive option doesn't seem to be deleting anything for some reason

### ISSUE THREAD

**Issue by arkhammodder** [Lyfhael/DeleteTweets#28]
Only the non archive version deletes tweets and gets rate limited after a bit of time

**Comment by tamalCodes** [comment]
I am not even getting a box or anything to upload the tweet.js file

**Comment by ToKroCZ** [comment]
Yeah, I can confirm that this method doesn't work. It says "Deletion complete" and lists like two retweets that it claims to have deleted, but it didn't. And it completely misses all normal tweets. The old method works fine.

**Comment by yaomtc** [comment]
Still not working. When I tried the archive method it would get stuck at 917 or something, the page would become unresponsive, I'd force close Chromium and then none of my tweets would be deleted.
